documentation/specs/src/interoperability/ethereum-bridge/transfers_to_ethereum.md

```diff
@@ -0,0 +1,135 @@
+# Transferring from Namada to Ethereum
+
+Moving assets from Namada to Ethereum will not be automatic, as opposed the
+movement of value in the opposite direction. Instead, users must send an
+appropriate transaction to Namada to initiate a transfer across the bridge
+to Ethereum. Once this transaction is approved, a "proof" will be created
+and posted on Namada.
+
+It is incumbent on the end user to  request an appropriate ["proof"](proofs.md) 
+of the transaction. This proof must be submitted to the appropriate Ethereum smart
+contract by the user to redeem Ethereum assets / mint wrapped assets. This also
+means all Ethereum gas costs are the responsibility of the end user.
+
+A relayer binary will be developed to aid users in accessing the proofs
+generated by Namada validators as well as posting this proof to Ethereum. It
+will also aid in batching transactions.
+
+## Moving value to Ethereum
+
+To redeem wrapped Ethereum assets, a user should make a transaction to burn
+their wrapped tokens, which the `#EthBridge` validity predicate will accept.
+
+Mints of a wrapped Namada token on Ethereum (including NAM, Namada's native token)
+will be represented by a data type like:
+```rust
+struct MintWrappedNam {
+    /// The Namada address owning the token
+    owner: NamadaAddress,
+    /// The address on Ethereum receiving the wrapped tokens
+    receiver: EthereumAddress,
+    /// The address of the token to be wrapped 
+    token: NamadaAddress,
+    /// The number of wrapped Namada tokens to mint on Ethereum
+    amount: Amount,
+}
+```
+
+If a user wishes to mint a wrapped Namada token on Ethereum, they must
+submit a transaction on Namada that:
+- stores `MintWrappedNam` on chain somewhere - TBD
+- sends the correct amount of Namada token to `#EthBridgeEscrow`
```

Comment:
```suggestion
To redeem wrapped Ethereum assets, a user should make a transaction to burn
their wrapped tokens, which the `#EthBridge` validity predicate will accept.
For sending NAM over the bridge, a user should send their NAM to 
`#EthBridgeEscrow`. In both cases, it's important that the user also adds a
`PendingTransfer` to the [Bridge Pool](#bridge-pool-validity-predicate).
```

I can spec this out further also but in short this will be simpler now that we will only handle NAM, and `PendingTransfer` replaces what was `MintWrappedNam`.